src/sdl2.nim

```diff
@@ -73,8 +73,9 @@ type
     timestamp*: uint32
     windowID*: uint32
     event*: WindowEventID
-    pad*: array[56-21, byte]
+    pad1,pad2,pad3: uint8
```

Comment:
Any reason why this is not `pad1: array[3, byte]` ?

Reply:
Not really, other types in this type section also have sequences of 3 bytes designated `pad1,pad2,pad3: uint8`, mainly because the SDL header also defines 3 uint8s in those places, and this is one of those places.